### PR TITLE
Feature/lga 3059 ci cd pipeline

### PR DESCRIPTION
**Pull request by TawneeOwl**
Create the circle CI pipeline and dockerfile for the case API

### CODE REVIEW

File: helm_deploy/laa-civil-case-api/templates/tests/test-connection.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (digitalali-moj) [inline helm_deploy/laa-civil-case-api/templates/tests/test-connection.yaml:1]: Is this being used anywhere? If not can we remove it?

File: requirements/generated/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,89 @@
+#
+# This file is autogenerated by pip-compile with Python 3.7
```
Comment (digitalali-moj) [inline requirements/generated/requirements.txt:2]: Are we using Python 3.7 or 3.9 like the docker image?

**Review by digitalali-moj — CHANGES_REQUESTED** [review]
This looks great Kyle. Some things to chat through when you get a sec, but really awesome stuff!

**Review by digitalali-moj — DISMISSED** [review]
Fantastic work!

**Review by Pheonnexx — APPROVED** [review]
LGTM - at least you caught those copy paste mistakes!

**Pull request merged**